### PR TITLE
READY: dev/using/data-types for Node.js

### PR DESCRIPTION
**Pull request by lukebakken**
OMG this document has *so many* examples.

Also, all examples can be run here:

https://github.com/basho/riak-nodejs-client-examples/blob/master/dev/using/data-types.js

### CODE REVIEW

File: source/languages/en/riak/dev/using/data-types.md [hunk 1]
```diff
@@ -197,6 +206,16 @@ var id = new RiakObjectId(bucketType, bucket, key);
 var counter = Client.DtFetchCounter(id);
 ```
 
+```javascript
+// The following can be passed as options to the *Counter methods on the
+// NodeJS Client object
```
Comment (laurenrother) [inline source/languages/en/riak/dev/using/data-types.md]: Should NodeJS change to Node.js here?

File: source/languages/en/riak/dev/using/data-types.md [hunk 2]
```diff
@@ -511,6 +613,17 @@ set = Set(bucket, key)
 var id = new RiakObjectId(bucket_type, bucket, key);
 ```
 
+```javascript
+// As with counters, with the Riak NodeJS Client you interact with sets on the
```
Comment (laurenrother) [inline source/languages/en/riak/dev/using/data-types.md]: Also here, should NodeJS change to Node.js?

File: source/languages/en/riak/dev/using/data-types.md [hunk 3]
```diff
@@ -2444,6 +3022,26 @@ Debug.WriteLine(format: "Context: {0}", args: Convert.ToBase64String(rslt.Contex
...
+    }
+
+    logger.info("context: '%s'", rslt.context.toString('base64'));
+});
+
+// Output:
+// context: 'g2wAAAACaAJtAAAACLQFHUmjDf4EYTBoAm0AAAAIxVKxC6F1L2dhSWo='
+```
+
+
```
Comment (laurenrother) [inline source/languages/en/riak/dev/using/data-types.md]: Why two spaces here?

File: source/languages/en/riak/dev/using/data-types.md (hunk 1 above)
Comment (lukebakken) [inline source/languages/en/riak/dev/using/data-types.md]: Yep! I'll get it.

**Comment by lukebakken** [comment]
@laurenrother all set :smile:

**Comment by laurenrother** [comment]
:cake:

**Pull request merged**